### PR TITLE
Update for version 1.1.3

### PR DESCRIPTION
**Pull request by cphyc**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,23 +9,19 @@ package:
...
 
 requirements:
   build:
     - {{ compiler('c') }}
     - rockstar-galaxies  # [linux]
   host:
-    - cython >=0.29.21,<3.0
-    - numpy
+    - cython >=3.0
+    - numpy >=1.17.5,<2.0
```
Comment (neutrinoceros) [inline recipe/meta.yaml:24]: I don't know yet how conda-forge will handle the ABI change in numpy 2 but the approach I'm seeing in other feedstocks (and yt's in particular) is to not pin here

File: recipe/meta.yaml [hunk 2]
```diff
@@ -8,23 +8,18 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/yt_astro_analysis-{{ version }}.tar.gz
-  sha256: 59307b598be8aab55b94e3b5a3dc185445885006b025fa0da50819e5c45297c0
-  # NOTE: Remove once https://github.com/yt-project/yt_astro_analysis/pull/219 is merged
-  # and shipped in a release.
-  patches:
-    - detect-rockstar.patch
```
Comment (neutrinoceros) [inline recipe/meta.yaml:15]: I think the patch file can be removed now ?

**Review by neutrinoceros — APPROVED** [review]

**Comment by cphyc** [comment]
@conda-forge-admin, please rerender

**Comment by cphyc** [comment]
Now ready to go :rocket:

**Pull request merged**